xls/dev_tools/check_header_guards.py

```diff
@@ -42,7 +43,8 @@ def check_header_guard(filepath, expected_guard):
     return False, None
 
 def find_h_files(repo_root):
-    # Find all .h files within the repo root, excluding xls/contrib and third_party
+    # Find all `.h` files within the repo root, excluding `xls/contrib` and
+    # `third_party`.
     h_files = []
     for root, _, files in os.walk(repo_root):
         if 'xls/contrib' in root or 'third_party' in root:
```

Comment:
Is or 'third_party' in root needed?  I don't think github has this directory.

Reply:
https://github.com/google/xls/tree/main/third_party